### ISSUE TITLE
Placeholder in 'Endpoint URL' field can be misleading

### ISSUE THREAD

**Issue by tdreid** [HarperDB/HarperDB_Studio#25]
Use of the placeholder 'localhost' in the login form's 'Endpoint URL' field can give the user the impression that `localhost` is a default that will be used if nothing else is entered here.

![image](https://user-images.githubusercontent.com/868675/41208303-0c1ccc64-6cf0-11e8-9aa1-7674396cfcff.png)

However if I enter nothing in this field, but otherwise provide my valid credentials and port number I will encounter the 'Missing Endpoint URL' message. In contrast if I type exactly 'localhost' in this field and leave all other settings the same my login is successful.

You may want to consider clarifying the placeholder. For example:

![image](https://user-images.githubusercontent.com/868675/41208379-c001c068-6cf0-11e8-9c37-3891ea6e8f7c.png)

**Comment by elipalmer** [comment]
Agreed, the whole thing is confusing, and made worse by adding http:// in front of the host causes a connection error.  I'm reviewing that bug now.

**Comment by elipalmer** [comment]
Created issue #28 to correct the bug of placing HTTP in front of the host causes a connection failure.  It will not correct the workflow above, only fix the bug.

**Comment by elipalmer** [comment]
I think we should have the text say what is described above, as well as have some text describing the port.  I don't think we should have default values in there at the moment.  I think the text should read:
Endpoint URL: Enter URL of host running HarperDB
Port: Enter PORT HarperDB is listening on

I'm open to different wording, as long as it gets the point across.
